### ISSUE TITLE
Remove adder-info debug output

### ISSUE THREAD

**Issue by eric-unc** [ac-freeman/adder-codec-rs#90]
See the following output:
```
Dimensions
	Width: 640
	Height: 360
	Color channels: 1
Source camera: FramedU8
ADΔER transcoder parameters
	Codec version: 2
	Time mode: AbsoluteT
	Ticks per second: 7650
	Reference ticks per source interval: 255
	Δt_max: 30600
File metadata
	File size: 83831246
	Header size: 33
	ADΔER event count: 9084179
	Events per pixel channel: 39
Calculating dynamic range...15%20336 0
25370 20336
27928 25370
28227 27928
28531 28227
28764 28531
Calculating dynamic range...16%28955 28764
Calculating dynamic range...20%31494 28955
32795 31494
33183 32795
33689 33183
Calculating dynamic range...21%34233 33689
Calculating dynamic range...23%35555 34233
35981 35555
36433 35981
Calculating dynamic range...29%40902 36433
42143 40902
42602 42143
42985 42602
43546 42985
Calculating dynamic range...30%43799 43546
44064 43799
Calculating dynamic range...32%44960 44064
45305 44960
Calculating dynamic range...33%45900 45305
46155 45900
46369 46155
Calculating dynamic range...34%46864 46369
47129 46864
47374 47129
Calculating dynamic range...43%51695 47374
52313 51695
52666 52313
52984 52666
53171 52984
Calculating dynamic range...44%53546 53171
53831 53546
Calculating dynamic range...47%55454 53831
Calculating dynamic range...49%55969 55454
Calculating dynamic range...54%56657 55969
Calculating dynamic range...55%57064 56657
Calculating dynamic range...73%66280 57064
68740 66280
69155 68740
Calculating dynamic range...77%69686 69155
Calculating dynamic range...79%70125 69686
71145 70125
71301 71145
Calculating dynamic range...81%72225 71301
72796 72225
Calculating dynamic range...83%73533 72796
73831 73533
Calculating dynamic range...86%74954 73831
75472 74954
Calculating dynamic range...88%75697 75472
Calculating dynamic range...90%75884 75697
Calculating dynamic range...96%78105 75884
79390 78105
80125 79390
Dynamic range                       
	Theoretical range:
		427.1653 dB (power)
		141.9012 bits
	Realized range:
		30.2141 dB (power)
		10.0369 bits
```

**Comment by eric-unc** [comment]
Also, what's up with the \r [here](https://github.com/ac-freeman/adder-codec-rs/blob/main/adder-info/src/main.rs#L134)?

**Comment by ac-freeman** [comment]
Carriage return

**Comment by eric-unc** [comment]
I know what it is, but why?

**Comment by ac-freeman** [comment]
So it prints/updates on one line, when that debugging output is removed again.

**Comment by ac-freeman** [comment]
Fixed #91
